.github/workflows/test_development_environment.yml

```diff
@@ -33,6 +33,8 @@ jobs:
             ${{ runner.os }}-build-${{ env.cache-name }}-	
             ${{ runner.os }}-build-	
             ${{ runner.os }}-
+      - name: Setup NPM Token
+        run: npm config set //npm.pkg.github.com/:_authToken ${{ secrets.GH_TOKEN_SUPPLYALLY_BOT }}
```

Comment:
`NPM_TOKEN` ENV thanks :)

Reply:
tried with `NPM_TOKEN`.. got [issue](https://github.com/rationally-app/mobile-application/runs/4655097475?check_suite_focus=true)... then used like that

Reply:
oh why?

Reply:
Thought all same like
```
- name: Setup NPM Token
  run: npm config set //npm.pkg.github.com/:_authToken $NPM_TOKEN
  env:
    NPM_TOKEN: ${{ secrets.GH_TOKEN_SUPPLYALLY_BOT }}
```

Reply:
ok error because of windows(`%NPM_TOKEN%`) vs linux shell(`$NPM_TOKEN`), am ok with this then